app/views/public_recipes/index.html.erb

```diff
@@ -3,7 +3,7 @@
 <% if @public_recipes.exists? %>
```

Comment:
- [ ] Upon running your app, I am getting the following error: 

![image](https://user-images.githubusercontent.com/58504655/227554400-c544cbb6-b493-4262-8e9d-5aac8e6bda42.png)

Reply:
@arslanbisharat , It would be better if you explain in detail about this problem , please ? 
In our case , we create the database and the app in running , it would be better if you can explain the problem that is causing that issue. thanks